scripts/singleton/compute.sh

```diff
@@ -28,7 +28,7 @@ rm -rf "$output_dir" # Delete possible remains from previous run
 mkdir -p "$output_dir"
 
 echo "Flattening source files into a single file"
-npx truffle-flattener contracts/RelayHub.sol > $output_dir/RelayHub.flattened.sol
+( echo "pragma experimental ABIEncoderV2;"; npx truffle-flattener contracts/RelayHub.sol |grep -v 'pragma experimental ABIEncoderV2') > $output_dir/RelayHub.flattened.sol
```

Comment:
could you explain what is going on here?

Reply:
It removes all occurrences of  "pragma experimental ABIEncoderV2;" and puts only one of it at the top of the file...